extras/src/main/java/com/google/gson/typeadapters/RuntimeTypeAdapterFactory.java

```diff
@@ -108,14 +108,14 @@
  *   shapeAdapter.registerSubtype(Circle.class, "Circle");
  *   shapeAdapter.registerSubtype(Diamond.class, "Diamond");
  * }</pre>
- * Finally, register the type adapter in your application's GSON builder:
+ * Finally, register the type adapter factory in your application's GSON builder:
  * <pre>   {@code
  *   Gson gson = new GsonBuilder()
- *       .registerTypeAdapter(Shape.class, shapeAdapter)
+ *       .registerTypeAdapterFactory(Shape.class, shapeAdapterFactory)
```

Comment:
Please!

On Wed, May 25, 2016 at 3:02 PM Juan Andrés <notifications@github.com>
wrote:

> In
> extras/src/main/java/com/google/gson/typeadapters/RuntimeTypeAdapterFactory.java
> <https://github.com/google/gson/pull/864#discussion_r64633095>:
>
> >   * <pre>   {@code
> >   *   Gson gson = new GsonBuilder()
> > - *       .registerTypeAdapter(Shape.class, shapeAdapter)
> > + *       .registerTypeAdapterFactory(Shape.class, shapeAdapterFactory)
>
> @JakeWharton <https://github.com/JakeWharton> You're right. That line
> should be .registerTypeAdapterFactory(shapeAdapterFactory).
>
> How would you rather have me summit the fix (since it's already been
> merged)? A new PR with this change only?
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/google/gson/pull/864/files/d4a9eb4e7bcbf0fa9e9e76c81fc86ff669f7c8ea#r64633095>
>